### DISCUSSION TITLE
Docker image location

### DISCUSSION THREAD

**Discussion by guppie70** [abiosoft/colima#587] [Q&A] [answered]
Hi all,

I have just switched from Docker Desktop to Colima on my new M1 Macbook Pro. I was able to set it up so that my AMD64 containers use Rosetta2 emulation and the speed increase over Docker Desktop is mind blowing. Happy new user here :-)
Am not too sure how Colima works, but can you please explain to me where I can find the location of the docker image files? Is it in the same location as Docker Desktop or somewhere inside the VM that Colima initiates when I start it up?
Regards,

Johan.

**Comment by abiosoft** [comment, marked as answer]
The config is located in `~/.colima`
The virtual machine is located in `~/.lima/colima`

The docker volumes and other data can be access inside the VM at `/var/lib/docker`
```
colima ssh -- sudo ls -lah /var/lib/docker
```

**Reply by guppie70** [reply]
Thanks so much for the quick response and the fantastic work that you are doing with this project!

**Reply by sspilleman** [reply]
@abiosoft Yesterday I installed colima (v0.6.9). According to this post I understand I should have a `~/.lima` directory, which doesn't exist on my mac. Is the current version using another directory?

**Reply by pgosar** [reply]
@sspilleman ~/.colima/_lima
